### PR TITLE
test: add GH actions for CI validations

### PR DESCRIPTION
**Pull request by manolo**

### CODE REVIEW

**Review by ZheSun88 — APPROVED** [review]

File: flow-tests/test-mixed/src/test/java/com/vaadin/flow/mixedtest/ui/IdTestIT.java [hunk 1]
```diff
@@ -26,6 +27,7 @@
 import com.vaadin.flow.testutil.ChromeBrowserTest;
 import com.vaadin.testbench.TestBenchElement;
 
+@Ignore("Fails in Containers")
```
Comment (Artur-) [inline flow-tests/test-mixed/src/test/java/com/vaadin/flow/mixedtest/ui/IdTestIT.java:30]: Why does it fail? We should not ignore it

It happens to be the only test that catches problems related to using a context path / servlet path

Comment (ZheSun88) [reply flow-tests/test-mixed/src/test/java/com/vaadin/flow/mixedtest/ui/IdTestIT.java:30]: agree... at least, there should be a ticket for further investigation.

**Review by ZheSun88 — CHANGES_REQUESTED** [review]
please check Artur's comment

File: flow-tests/test-mixed/src/test/java/com/vaadin/flow/mixedtest/ui/IdTestIT.java (hunk 1 above)
Comment (manolo) [reply flow-tests/test-mixed/src/test/java/com/vaadin/flow/mixedtest/ui/IdTestIT.java:30]: Opened this ticket and assigned to me https://github.com/vaadin/flow/issues/12458

**Review by ZheSun88 — APPROVED** [review]

**Pull request merged**